themes/default/content/docs/guides/pulumi-packages/how-to-author.md

```diff
@@ -73,6 +73,11 @@ Metadata for your package is generated from the `schema.json` in your repository
   - `category/CATEGORY`: replace `CATEGORY` with one of `cloud`, `database`, `infrastructure`, `monitoring`, `network`, `utility`, `versioncontrol`
   - `kind/KIND`: replace `KIND` with one of `native`, `component`
     - Note: don't set a kind if you're bridging a Terraform provider
+- `pluginDownloadURL`: a web-accessible URL that contains the compiled binary plugin associated with your package; for more information see [Publish your package]({{<relref "#publish-your-package">}})
```

Comment:
Since this is just an anchor, you can use that (unlike links, Hugo doesn't validate that anchors are correct):

```suggestion
- `pluginDownloadURL`: a web-accessible URL that contains the compiled binary plugin associated with your package; for more information see [Publish your package](#publish-your-package).
```